hx_lti_initializer/templates/hx_lti_initializer/index_button.html

```diff
@@ -0,0 +1 @@
+<a role="button" href="index_view"><div class="btn btn-default">Return to Index</div></a>
```

Comment:
This should use the django **url** [template tag](https://docs.djangoproject.com/en/1.8/ref/templates/builtins/) to generate the correct URL. Ex:

```python
<a role="button" href="{% url 'index_view' %}"><div class="btn btn-default">Return to Index</div></a>
```